integration/suite.go

```diff
@@ -0,0 +1,114 @@
+package integration
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/medibloc/panacea-doracle/crypto"
+	"log"
+	"path/filepath"
+	"time"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
+	suite.Suite
+
+	initScriptDir      string
+	initScriptFilename string
+	initScriptEnvs     []string
+
+	dktPool     *dockertest.Pool
+	dktResource *dockertest.Resource
+}
+
+func NewTestSuite(initScriptAbsPath string, initScriptEnvs []string) TestSuite {
+	if !filepath.IsAbs(initScriptAbsPath) {
+		log.Panicf("path must be absolute: %s", initScriptAbsPath)
+	}
+	dir, filename := filepath.Split(initScriptAbsPath)
+
+	return TestSuite{
+		initScriptDir:      dir,
+		initScriptFilename: filename,
+		initScriptEnvs:     initScriptEnvs,
+	}
+}
+
+func (suite *TestSuite) SetupSuite() {
+	dktPool, err := dockertest.NewPool("")
+	if err != nil {
+		log.Panicf("Could not connect to docker: %v", err)
+	}
+	suite.dktPool = dktPool
+}
+
+func (suite *TestSuite) SetupTest() {
+	var err error
+
+	suite.dktResource, err = suite.dktPool.RunWithOptions(
+		&dockertest.RunOptions{
+			Repository: "ghcr.io/medibloc/panacea-core",
+			Tag:        "master",
```

Comment:
This docker image can be found in the https://github.com/medibloc/panacea-core/pkgs/container/panacea-core (published by https://github.com/medibloc/panacea-core/blob/master/.github/workflows/docker-publish.yml)